src/tests/Microsoft.PowerFx.Core.Tests.Shared/TexlTests.cs

```diff
@@ -2693,6 +2693,25 @@ public void TexlFunctionTypeSemanticsTable_Negative(string script, string expect
                 features: Features.PowerFxV1);
         }
 
+        [Theory]
+        [InlineData("Trace(\"hello\")")]
+        [InlineData("Trace(\"hello\", TraceSeverity.Warning)")]
+        [InlineData("Trace(\"hello\", TraceSeverity.Warning, { a: 1 })")]
+        public void TexlFunctionTypeSemanticsTrace(string expression)
+        {
+            foreach (var powerFxV1 in new[] { false, true })
+            {
+                var features = powerFxV1 ? Features.PowerFxV1 : Features.None;
+                var engine = new Engine(new PowerFxConfig(features));
+                var options = new ParserOptions() { NumberIsFloat = true, AllowsSideEffects = true };

```

Comment:
>NumberIsFloat = true [](http://example.com/codeflow?start=52&length=20)

Do we need this parse option to run the test?

Reply:
Good point, it's not necessary. Removed.